src/CollectdWinService/config/WriteHttpPluginConfig.cs

```diff
@@ -54,6 +54,27 @@ public int MaxIdleTime
                 set { base["MaxIdleTime"] = value; }
             }
 
+            [ConfigurationProperty("UserName", IsRequired = false)]
+            public string UserName
+            {
+                get { return (string)base["UserName"]; }
+                set { base["UserName"] = value; }
+            }
+
+            [ConfigurationProperty("Password", IsRequired = false)]
+            public string Password
+            {
+                get { return (string)base["Password"]; }
+                set { base["Password"] = value; }
+            }
+
+            [ConfigurationProperty("UseSafeNames", IsRequired = false)]

```

Comment:
that is right, if we do it as part of WindowsPerformanceCounter, it will be a global replace. Lets then do it as part of WriteHttp.config under node.